recipe/meta.yaml

```
@@ -1,66 +1,62 @@
 {% set name = "statsmodels" %}
-{% set version = "0.13.5" %}
+{% set version = "0.14.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/statsmodels-{{ version }}.tar.gz
-  sha256: 593526acae1c0fda0ea6c48439f67c3943094c542fe769f8b90fe9e6c6cc4871
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6875c7d689e966d948f15eb816ab5616f4928706b180cf470fd5907ab6f647a4
 
 build:
-  number: 1
-  skip: true  # [py<37] 
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   build:
     - {{ compiler('c') }}
   host:
     - python
+    - cython 0.29.32
```

```suggestion
    - cython >=0.29.26,<3
```
There is no advantage in pinning cython to the lowest minor.